tools/windows-node-installer/pkg/cloudprovider/factory.go

```diff
@@ -70,9 +70,9 @@ func CloudProviderFactory(kubeconfigPath, credentialPath, credentialAccountID, r
 
 	switch provider := cloudProvider.Type; provider {
 	case v1.AWSPlatformType:
-		return aws.New(oc, credentialPath, credentialAccountID, resourceTrackerFilePath)
+		return aws.New(oc, imageID, instanceType, sshKey, credentialPath, credentialAccountID, resourceTrackerFilePath)
```

Comment:
need to fix this.

Reply:
Not sure what you need to fix but please track it in Jira.

Reply:
this is a result of mistake that occurred during the rebase, I selected my change initially and later realised it and restored it to how originally it looks like.  I don't think we need to track anything in Jira.